core/templates/dev/head/base.html

```diff
@@ -255,10 +255,20 @@
                           <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
                             <create-activity-button></create-activity-button>
 
-                            <li class="oppia-clickable-navbar-element pull-right">
-                              <a class="oppia-navbar-tab" href="/donate" translate="I18N_TOPNAV_DONATE">
+                          <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
+                            <div class="pull-right oppia-navbar-button-container">
+                              <a href="/donate" class="btn oppia-navbar-button oppia-navbar-hide-on-small-width protractor-test-create-activity oppia-transition-200"
+                                <span translate="I18N_TOPNAV_DONATE"></span>
                               </a>
-                            </li>
+                            </div>
+                          </ul>
+                          <ul ng-if="!windowIsNarrow" class="nav oppia-navbar-tabs">
```

Comment:
Ah, I missed that, thanks.

It's actually better to have just one clause, and use ng-class, viz.:

`<ul class="nav" ng-class="{'oppia-navbar-tabs-narrow': windowIsNarrow, 'oppia-navbar-tabs': !windowIsNarrow}"`